src/test/resources/responses/getATB.ttl

```diff
@@ -25,4 +25,5 @@
                                  skos:notation      "994686011" ;
                                  adms:schemaAgency  "Brønnøysundregistrene"
                                ] ;
+        foaf:homepage          "www.atb.no" ;
```

Comment:
Range til foaf:homepage er foaf:Document, så denne skal vere ei lenke, feks slik:
```
foaf:homepage          <https://www.atb.no> ;
```

Reply:
Vi får sjelden, eller aldri, en uri fra enhetsregisteret som vil bli godtatt som 'well-formed'

Reply:
Skal vi alltid bare legge til `https://` og satse på at det er korrekt?

Reply:
Skal den endringen kun bli gjort for RDF, eller skal også json svare med endret verdi?

Reply:
Eg har oppdatert #99 med presisering av dette:
> Vi forsøker å sette scheme = http. Om det ikkje resulterer i ein gyldig URI, droppar vi egenskapen for denne enheten.

Reply:
Gjelder kun for RDF. I json-respons kan vi legge inn verdien brreg har.

Reply:
234294b